app/views/groups/show.html.erb

```diff
@@ -17,8 +17,8 @@
           <%= @viewmodel.leader_info_button %>
           <%= @viewmodel.join_group_button %>
           <%= @viewmodel.withdraw_membership_button %>
-          <%= @viewmodel.new_evaluation_button %>
-          <%= @viewmodel.resort_leader_evaluation_button %>
+          <%#= @viewmodel.new_evaluation_button %>
```

Comment:
Can you remove the commented line please

Reply:
I removed it!